tests/node/sourcemap-test.js

```diff
@@ -0,0 +1,27 @@
+var fs = require('fs');
+
+QUnit.module('sourcemap validation', function() {
+  var assets = ['ember.debug', 'ember.prod', 'ember.min'];
+
+  assets.forEach(asset => {
+    QUnit.test(`${asset} has only a single sourcemaps comment`, function(assert) {
+      var jsPath = `dist/${asset}.js`;
+      assert.ok(fs.existsSync(jsPath));
```

Comment:
The node tests are failing because this apparently runs in debug builds (not prod builds) so `.prod.js` and `.min.js` files are not present.

Can you tweak https://github.com/emberjs/ember.js/blob/8324f807e72f7facc258f8afa785eec6e993b532/.travis.yml#L86-L87 to:

```yml
- env:
  - TEST_SUITE=node
  - EMBER_ENV=production
  node_js: "6"
```